sklearn/tests/test_dummy.py

```diff
@@ -617,3 +617,49 @@ def test_dummy_regressor_on_nan_value():
     clf.fit(X, y)
     y_pred = clf.predict(X)
     assert_array_equal(y_pred, y_expected)
+
+
+def test_dummy_regressor_on_object_values():
+    X = np.array([[], [], []], dtype=object)
```

Comment:
too much copy-paste :)

Reply:
Maybe not using zero feature array here is better. I prefer the previous version.